app/js/controllers/main.js

```diff
@@ -3,12 +3,11 @@
 export default class MainController extends Controller {
   constructor(options) {
     super(options);
-    this.attached = false;
-    this.waitToBeOpened();
+    this._waitToBeOpened();
   }
 
-  waitToBeOpened() {
-    var openGesture = {
+  get openGesture() {
```

Comment:
If we did that there would basically be nothing at all left in MainController, I think.  I don't have a strong opinion except that that kind of refactoring seems out of scope here. So I'll leave this to you or Justin to do in a separate pass if you want to.